l10n_nl_tax_declaration_reporting/i18n/nl.po

```diff
@@ -0,0 +1,314 @@
+msgstr "Dit menu drukt een BTW-aangifte af. Selecteer één of meer periodes uit het fiscale jaar. De benodigde informatie voor de BTW-aangifte wordt door Odoo automatisch verzameld op basis van de facturen. De rapportage geeft de actuele stand van zaken, zodat u op elk moment kunt zien hoeveel belasting u schuldig bent aan het begin en eind van een periode."
+
+#. module: l10n_nl_tax_declaration_reporting
+#: report:account.vat.declaration.nl:0
+msgid "Pre-tax"
+msgstr "5b Voorbelasting"
+
+#. module: l10n_nl_tax_declaration_reporting
+#: report:account.vat.declaration.nl:0
+msgid "4b Deliveries/services of countries in the EU"
+msgstr "4b Leveringen/diensten uit landen binnen de EU"
+
+#. module: l10n_nl_tax_declaration_reporting
+#: report:account.vat.declaration.nl:0
+msgid "1a Deliveries/services taxed with 21%"
+msgstr "1a Leveringen/diensten belast met 21%"
+
+#. module: l10n_nl_tax_declaration_reporting
+#: selection:account.vat.declaration.nl,based_on:0
+msgid "Invoices"
+msgstr "Facturen"
+
+#. module: l10n_nl_tax_declaration_reporting
+#: model:ir.actions.report.xml,name:l10n_nl_tax_declaration_reporting.report_account_vat_declaration_nl
+msgid "VAT Report NL"
+msgstr "BTW Rapport NL"
+
+#. module: l10n_nl_tax_declaration_reporting
+#: field:account.vat.declaration.nl,fiscalyear_id:0
+#: report:account.vat.declaration.nl:0
+msgid "Fiscal Year"
+msgstr "Boekjaar"
+
+#. module: l10n_nl_tax_declaration_reporting
+#: field:account.vat.declaration.nl,target_move:0
+msgid "Target Moves"
+msgstr "Doelrekening"
+
+#. module: l10n_nl_tax_declaration_reporting
+#: report:account.vat.declaration.nl:0
+msgid "5c Subtotal to pay (+) or to reclaim (-)"
+msgstr "5c Subtotaal te betalen (+) of terug te vragen (-)"
+
+#. module: l10n_nl_tax_declaration_reporting
+#: field:account.vat.declaration.nl,based_on:0
+msgid "Based on"
+msgstr "Gebaseerd op"
+
+#. module: l10n_nl_tax_declaration_reporting
+#: view:account.vat.declaration.nl:0
+#: selection:account.vat.declaration.nl,filter:0
+#: report:account.vat.declaration.nl:0
+msgid "Periods"
+msgstr "Periodes"
+
+#. module: l10n_nl_tax_declaration_reporting
+#: view:account.vat.declaration.nl:0
+msgid "Cancel"
+msgstr "Annuleer"
+
+#. module: l10n_nl_tax_declaration_reporting
+#: report:account.vat.declaration.nl:0
+msgid "Based On"
+msgstr "Gebaseerd op"
+
+#. module: l10n_nl_tax_declaration_reporting
+#: field:account.vat.declaration.nl,date_from:0
+msgid "Start Date"
+msgstr "Startdatum"
+
+#. module: l10n_nl_tax_declaration_reporting
+#: help:account.vat.declaration.nl,chart_tax_id:0
+msgid "Select Charts of Taxes"
+msgstr "Kies Taksstructuur"
```

Comment:
Finding 'Taks' slightly odd as a Northener ;-) Tax chart in the Odoo accounting core module is translated as 'Belastingschema'. Would you consider this terminology instead?